### DISCUSSION TITLE
Google Photo images and videos

### DISCUSSION THREAD

**Discussion by joomleb** [mpratt/Embera#83] [Ideas]
Hi guys, 
Please, Can you consider to add support to Google Photo images, videos URLs ?
([Here](https://sites.google.com/a/gapps.uwcsea.edu.sg/dover-primary-ict/resources/grabasharelinkfromgoogle) an old guide), but also support to Albums ([here a module example](https://joomlakave.com/joomla-modules/google-photo-album-display-for-joomla)).

**Comment by mpratt** [comment]
Hi @joomleb 

Thanks for the suggestion!
You are right, it would be nice to support Google Photo Images aswell as the other providers you suggested on your other thread.
However, correct me if I'm wrong, I don't think either of those services provide an oembed endpoint.

The scope of Embera is to be a simple oembed consumer that provides an interface to embed data from multiple providers on your projects. As long as Google / Wikipedia don't provide an official oembed endpoint, Embera cannot support them officially.

You could get away by creating a new provider and using the `fakeResponse()` method to provide an embeddable html response from any of your providers. Another approach is to create a provider for [Embedly](https://docs.embed.ly/docs/oembed) which supports Wikipedia and returns an oembed response.

That's the best I can suggest but for now, those providers are out of the scope of this library. :disappointed:
